### PR TITLE
example of using updated SupersetAPI policies in cocosign

### PR DESCRIPTION
**Pull request by cppvik**
Rego policy updated to support the new version of superset policy api

### CODE REVIEW

File: .valint_pass.yaml [hunk 1]
```diff
@@ -16,94 +16,19 @@ attest:
...
+          signed: false
           format: attest-cyclonedx-json
           rego:
+            path: .valint.rego
             args:
               superset:
-                username: SUPERSET_USERNAME
-                password: SUPERSET_PASSWORD
-                env: dev
+                username: SMZ5qKFXMg7oe6ZmJjRv4BHzxXEKdqmX
```
Comment (eram) [inline .valint_pass.yaml:25]: Not creds in code pls

File: .valint.rego [hunk 2]
```diff
@@ -1,67 +1,41 @@
...
 allow  {
-  policy.unmaintained.allow
   policy.cve.allow
-  policy.images.allow
-  policy.licences.allow
-}
-
-errors[msg] {
-  msg := policy.unmaintained.errors[_]
+  # policy.images.allow
```
Comment (eram) [inline .valint.rego:8]: Do we need comments in code?

**Review by eram — CHANGES_REQUESTED** [review]

File: .valint.rego (hunk 2 above)
Comment (cppvik) [reply .valint.rego:8]: cocosign is able to support multiple built-in policies on superset, but right now only one of them is working and documented. These comments show how to run multiple built-in policies in the future.

File: .valint_pass.yaml (hunk 1 above)
Comment (cppvik) [reply .valint_pass.yaml:25]: sure, sorry

**Comment by cppvik** [comment]
To pass the tests we need to merge [this](https://github.com/scribe-security/cocosign/pull/56) first.

**Pull request merged**